### PR TITLE
PWGHF code for the J/psi to mumu channel at midrapidity in ALICE3

### PR DESCRIPTION
**Pull request by urasantonio**
Relevant for ALICE3 only

### CODE REVIEW

**Comment by ginnocen** [comment]
Thanks @urasantonio, we will look at it. And merge them in the right order. Thanks!

**Comment by ginnocen** [comment]
hi @urasantonio, it seems there are some conflicts. Could you try to rebase on the latest dev version? Thanks

**Comment by iarsene** [comment]
Hi @urasantonio, @ginnocen,
In this PR there seem to be many changed files that are unrelated to this PR. For example, the addition of comments at the end of each table entry, which was done by Paul already. Please check.

**Comment by ginnocen** [comment]
@iarsene, not yet ready for review. It was turned to non-draft for testing the CI. But I set it back to Draft. Thanks

**Comment by vkucera** [comment]
@urasantonio @ginnocen Convert to draft please. Review needed.

File: Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h [hunk 1]
```diff
@@ -279,11 +282,19 @@ auto EJpsi(const T& candidate)
...
 }
 
+// Jpsi → e+e-
 template <typename T>
 auto InvMassJpsiToEE(const T& candidate)
 {
   return candidate.m(array{RecoDecay::getMassPDG(kElectron), RecoDecay::getMassPDG(kElectron)});
 }
+
+// Jpsi → mu+mu-
```
Comment (vkucera) [inline Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h:292]: Use proper characters.

File: Analysis/Tasks/PWGHF/CMakeLists.txt [hunk 2]
```diff
@@ -88,8 +93,13 @@ o2_add_dpl_workflow(hf-task-lc
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing
                     COMPONENT_NAME Analysis)
 
-o2_add_dpl_workflow(hf-task-jpsi
-                    SOURCES taskJpsi.cxx
+o2_add_dpl_workflow(hf-task-jpsi-toee
+                    SOURCES taskJpsiToEE.cxx
```
Comment (vkucera) [inline Analysis/Tasks/PWGHF/CMakeLists.txt:97]: @ginnocen Didn't we want to have one task per hadron, that would include all considered decay channels?

Comment (ginnocen) [reply Analysis/Tasks/PWGHF/CMakeLists.txt:97]: in principle yes. After your PID refactory we should be able to merge everything together.

**Comment by vkucera** [comment]
Keeping as draft until merged with the e e channel code

**Pull request closed without merging**